shprofile.sh

```diff
@@ -51,6 +56,28 @@ shpRequiredProfile=''
 # All prefixed with 'shp' to facilitate their reuse #
 # in profile's shell scripts                        #
 #####################################################
+split() { # args: string delimiter result_var
```

Comment:
This function is never used
In addition, this function should be removed of the user context at the end of `shprofile` execution by using `shpClearEnvironment`